dev/tools/vitool/lib/vitool.dart

```diff
@@ -8,7 +8,7 @@ import 'dart:math';
 import 'package:collection/collection.dart';
 import 'package:meta/meta.dart';
 import 'package:vector_math/vector_math_64.dart';
-import 'package:xml/xml.dart' hide parse;
+import 'package:xml/xml.dart';
```

Comment:
I missed this in a previous patch - parse is deprecated and not used in this file anymore.